RockWeb/Themes/NewSpring/assets/Lava/LAYOUT/editorial-detail.lava

```diff
@@ -248,47 +210,41 @@
       "url": "https://newspring.cc/GetImage.ashx?id=661329"
     }
   },
-  "author": "{% if communicatorNames and communicatorNames != empty and communicatorNames != '' %}{{ communicatorNames | Trim }}{% else %}{{ 'Global' | Attribute:'OrganizationName' }}{% endif %}",
+  "author": {
+    "@type": "{% if communicators and communicators != empty and communicators != '' %}Person{% else %}Organization{% endif %}",
+    "name": "{% if communicators and communicators != empty and communicators != '' %}{{ communicators | Trim }}{% else %}{{ 'Global' | Attribute:'OrganizationName' }}{% endif %}"
+  },
```

Comment:
You're using the same `if` here twice. Would it make sense to move the `if` outside and just return two different `author` objects depending on the result of the `if`? That way it doesn't have to do the check twice?

Reply:
I guess you're okay with this?